chord_metadata_service/discovery/fields.py

```diff
@@ -338,22 +329,34 @@ def filter_queryset_field_value(qs: QuerySet, field_props, value: str):
     the `mapping` value is based on the same model as the queryset.
     """
 
-    model, field = get_model_and_field(
+    model, field = f_utils.get_model_and_field(
         field_props["mapping_for_search_filter"] if "mapping_for_search_filter" in field_props
         else field_props["mapping"]
     )
 
+    group_by = field_props.get("group_by")
+
     if field_props["datatype"] == "string":
-        condition = {f"{field}__iexact": value}
+        if group_by:
```

Comment:
missing tests for this - is it relatively straightforward, or too complex to bother?